python/lsst/resources/_resourcePath.py

```diff
@@ -259,7 +297,7 @@ def __new__(
                 )
 
             parsed, dirLike = subclass._fixupPathUri(
-                parsed, root=root, forceAbsolute=forceAbsolute, forceDirectory=forceDirectory
+                parsed, root=root_uri, forceAbsolute=forceAbsolute, forceDirectory=forceDirectory
```

Comment:
If `parsed` is already joined with `root_uri`, should `root_uri` be set to `None` before sending it to this method?

Reply:
This is working because `root` parameter will be ignored because the path is now an absolute path and also the subclass is not a file URI scheme any more. I agree it would be clearer to set it to None once we've used it above.